Civi/Financeextras/Setup/Manage/AbstractManager.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Civi\Financeextras\Setup\Manage;
+
+/**
+ * Describes the interface for entities that are
+ * to be managed (created, removed ..etc) during the
+ * extension installations, disabling ..etc.
+ *
+ */
+abstract class AbstractManager {
```

Comment:
I think Abstract Manager is very vague. Perhaps `AbstractSetupManager `might be better? As if we refer to Drupal https://www.drupal.org/docs/develop/standards/object-oriented-code#naming, since we use drupal + civicrm coder as phpcs standard, class name should be as 

```
Classes and interfaces should have names that stand alone to tell what they do without having to refer to the namespace, read well, and are as short as possible without losing functionality information or leading to ambiguity.
```